hooks/scene_operation_tk-houdini.py

```diff
@@ -72,13 +75,13 @@ def execute(
         elif operation == "open":
             # give houdini forward slashes
             file_path = file_path.replace(os.path.sep, "/")
-            hou.hipFile.load(six.ensure_str(file_path))
+            hou.hipFile.load(sgutils.ensure_str(file_path))
```

Comment:
I'm wondering if we can use just `str` or no extra function given that `file_path` is already a string (probably).